src/main/java/teammates/common/datatransfer/InstructorPrivileges.java

```diff
@@ -132,24 +132,24 @@ public InstructorPrivileges() {
     public InstructorPrivileges(String instrRole) {
         this();
         switch (instrRole) {
-        case Const.InstructorPermissionRoleNames.INSTRUCTOR_PERMISSION_ROLE_COOWNER:
-            setDefaultPrivilegesForCoowner();
-            break;
-        case Const.InstructorPermissionRoleNames.INSTRUCTOR_PERMISSION_ROLE_MANAGER:
-            setDefaultPrivilegesForManager();
-            break;
-        case Const.InstructorPermissionRoleNames.INSTRUCTOR_PERMISSION_ROLE_OBSERVER:
-            setDefaultPrivilegesForObserver();
-            break;
-        case Const.InstructorPermissionRoleNames.INSTRUCTOR_PERMISSION_ROLE_TUTOR:
-            setDefaultPrivilegesForTutor();
-            break;
-        case Const.InstructorPermissionRoleNames.INSTRUCTOR_PERMISSION_ROLE_CUSTOM:
-            setDefaultPrivilegesForCustom();
-            break;
-        default:
-            setDefaultPrivilegesForCustom();
-            break;
+            case Const.InstructorPermissionRoleNames.INSTRUCTOR_PERMISSION_ROLE_COOWNER:
```

Comment:
Hm we might need to update the teammates java style doc on the `case` indentation as it currently specifies
```
switch (condition) {
case ABC:
    statements;
    // etc...
```
:stuck_out_tongue: